sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusScaleMonitor.cs

```diff
@@ -11,10 +11,13 @@
 using Azure.Messaging.ServiceBus;
 using Azure.Messaging.ServiceBus.Administration;
 using Microsoft.Azure.WebJobs.Extensions.ServiceBus.Config;
+using Microsoft.Extensions.Options;
+using System.Xml.Serialization;
 
 namespace Microsoft.Azure.WebJobs.ServiceBus.Listeners
 {
-    internal class ServiceBusScaleMonitor : IScaleMonitor<ServiceBusTriggerMetrics>
+    //internal class ServiceBusScaleMonitor : IScaleMonitor<ServiceBusTriggerMetrics>
+    internal class ServiceBusScaleMonitor : ITargetScaleMonitor<ServiceBusTriggerMetrics>
```

Comment:
Wait on finalizing changes in this PR before merging https://github.com/Azure/azure-sdk-for-net/pull/30639/files